src/main/java/duke/ui/UI.java

```diff
@@ -0,0 +1,83 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+/**
+ * Display class duke.ui.UI for duke chatbot & handle user input
+ */
+public class UI {
```

Comment:
Some of the contents of class Duke are redundant with the contents of this class. Consider removing the redundant variables & methods from Duke.